README.md

```diff
@@ -5,7 +5,6 @@
 # 📗 Table of Contents
 
 - [📗 Table of Contents](#-table-of-contents)
-- [📖 Space Mission App ](#-space-mission-app-)
   - [🛠 Built With ](#-built-with-)
```

Comment:
![image](https://user-images.githubusercontent.com/98453979/225403334-e6f0ace4-50ee-460c-9f8c-f527a60493e2.png)

- [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).
- [ ] The link to your `MIT` License raises a `404` page error when visited, kindly adjust the pathname to make your readme more professional